lib/widgets/screens/sign_up/sign_up_business_help_selection.dart

```diff
@@ -17,27 +20,51 @@ class SignupBusinessHelpSelectionScreen extends StatefulWidget {
 
 class _SignupBusinessHelpSelectionScreenState
     extends State<SignupBusinessHelpSelectionScreen> {
+  late final ContentProvider _contentProvider;
+  late final List<SelectChip> _expertiseChips;
+
+  @override
+  void initState() {
+    super.initState();
+    _contentProvider = Provider.of<ContentProvider>(context, listen: false);
+    _expertiseChips = _contentProvider.expertiseOptions!
```

Comment:
There only seem to be a few expertises displayed here, is that the expected behavior?

Not sure if we were supposed to show all industries/expertises in the onboarding flow

Reply:
The UI flow is hardcoded to show pages for the mentor, so you are not seeing the pages I worked with. I did not want to change that to prevent conflicts with Rupal's work. It is a bit messy right now but trust me we will clean this up asap